sdk/core/azure-core/src/http/logging_policy.cpp

```diff
@@ -77,12 +77,11 @@ std::unique_ptr<RawResponse> Azure::Core::Http::LoggingPolicy::Send(
     Request& request,
     NextHttpPolicy nextHttpPolicy) const
 {
-  if (Logging::Details::ShouldWrite(LogClassification::Request))
+  if (Logging::Internal::ShouldLog(Logging::LogLevel::Verbose))
   {
-    Logging::Details::Write(LogClassification::Request, GetRequestLogMessage(request));
+    Logging::Internal::Log(Logging::LogLevel::Verbose, GetRequestLogMessage(request));
   }
-
-  if (!Logging::Details::ShouldWrite(LogClassification::Response))
+  else
```

Comment:
Is this really the logic we want?

Reply:
From offline discussion, yes it is. We are still calling the next policy outside the else block, below.